### DISCUSSION TITLE
Console polluted with [Fast Refresh] done/rebuilding

### DISCUSSION THREAD

**Discussion by balazsorban44** [vercel/next.js#13001] [Help] [answered]
![image](https://user-images.githubusercontent.com/18369201/82139994-9db3a300-982c-11ea-93ad-95c26cf4b5ff.png)

Is it possible to turn this off somehow? If it is a bug, maybe a `console.clear()` between rebuilds would help?

It appeared when I upgraded to Next.js 9.4.1

**Comment by sambhav-gore** [comment, marked as answer]
As an interim solution, you can hide the messages coming from the hot reloader by `right-click` and select `hide messages from .... ` , see below:

![hide-hot-dev](https://user-images.githubusercontent.com/11697969/82150287-6f4bbd00-9858-11ea-8e8b-290f495bd1cd.png)

**Reply by balazsorban44** [reply]
yes, actually found it right after. I guess it is good enough for me as an answer, thanks!
